common/components/ui/Input.scss

```diff
@@ -81,7 +81,7 @@
       color: $input-color-placeholder;
     }
     &:not([disabled]):not([readonly]) {
-      &.invalid.has-blurred.has-value {
+      &.invalid.has-blurred {
```

Comment:
We actually want the red border even when the field hasn't yet blurred, as it's a common pattern for users to be stuck and not realize they have to manually fill out nonce while offline.

@wbobeirne Any ideas on the best pattern here that won't affect all other inputs? Nonce is really a special case

Reply:
Seems to me the only fix here is a separate class and prop for this, since we need different functionality in different cases. Something like `showErrorWhileEmpty`, so that the selector is:
```
&.invalid.has-blurred.has-value,
&.invalid.has-blurred.show-error-empty {
  // ...
}
```

Alternatively, we could just make it so that the invalid boolean is always shown when passed, and change cases where we don't want it to while it doesn't have value to not pass it. I think both approaches are valid.